src/commands/check/checkGlobal.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-console */
-import { execa, execaCommand } from 'execa'
+import { async as ezspawnAsync } from '@jsdevtools/ez-spawn'
```

Comment:
```suggestion
import { async as execute } from '@jsdevtools/ez-spawn'
```

A bit nitpicking, but how about just making the name easy by describing what it does than what it is made?

I don't even mind NTR it to be `execa` 🤣